legion/engine/physics/halfedgeedge.hpp

```diff
@@ -54,6 +55,15 @@ namespace legion::physics
 		{
 			return nextEdge->edgePosition - edgePosition;
 		}
+
+        void DEBUG_drawEdge(const math::mat4& transform, const math::color& debugColor,float time = 20.0f, float width = 5.0f)
```

Comment:
is DEUG in caps necessary? Same as with DEBUG_drawFace this could maybe be implemented as debug::drawEdge as well for convenience. This is not a must though